projects/schematics/src/shared/utils/file-utils.ts

```diff
@@ -663,7 +672,10 @@ function getParamName(
   }
 
   for (const constructorParameter of constructorParameters) {
-    if (constructorParameter.getText().includes(classType.className)) {
+    // constructor parameter text is e.g. of form 'utilsService: UtilsService'
+    // we don't want to find occurences like otherParam: OtherUtilsService, 
+    // therefore we add blank before the class name
```

Comment:
I don't quite understand this comment?

Reply:
I provided a better solution that properly uses the ts syntax tools. Does not need such a comment then